### DISCUSSION TITLE
[Feature] Do not jump immediately to full-screen photo from the Map view circle showing the number of photos taken in a given location. Show thumbnails instead

### DISCUSSION THREAD

**Discussion by dgtal1** [immich-app/immich#8886] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I browse photos from the Map view and I see the circles with numbers - I click a circle and I'm immediately taken to some photo from this location:
![image](https://github.com/immich-app/immich/assets/27864579/ebdb1ecb-8cef-4caf-8789-d9deca111c44)

I don't think it's a convenient way to browse photos from this location I clicked. Instead I like how Photoprism solved it - they show thumbnails for this location under the map and then I'm able to pick a photo I want to start with from the grid of photos:
![image](https://github.com/immich-app/immich/assets/27864579/b3dca7f4-fd0e-4f8e-a24b-715b76da0db0)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4519
